dashboard-frontend/public/index.html

```diff
@@ -26,10 +26,108 @@
     -->
     <title>React App</title>
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+    <style>
```

Comment:
It's fine that this is in the base html when the site is built, but can't it be included as a fragment somehow? That way we can reuse it? Not urgen. 

But it's very cool looking.

Reply:
I just read the create-react-app docs (https://create-react-app.dev/) and didn't saw that they offer this out of the box.

It can be archived by taking advantage of npm scripts or by ejecting from create-react-app and using a Webpack plugin.

Since these styles and the html markup are only to replicate the LoaderPage, I don't think it's a big problem maintain these two in sync, since it's not something that changes often.

A cheap improvement that I can do at some point is to update the LoaderPage to use the style declarations defined here, making more explicit the relation between the two. I can also write a section about this in the README.md when I create it.